components/raftstore/src/store/worker/pd.rs

```diff
@@ -1290,6 +1334,35 @@ where
         };
         self.remote.spawn(f);
     }
+
+    fn handle_region_cpu_records(&mut self, records: Arc<CpuRecords>) {
+        calculate_region_cpu_records(self.store_id, records, &mut self.region_cpu_records)
+    }
+}
+
+fn calculate_region_cpu_records(
+    store_id: u64,
+    records: Arc<CpuRecords>,
+    region_cpu_records: &mut HashMap<u64, u32>,
+) {
+    for (tag, ms) in &records.records {
+        let record_store_id = tag.infos.store_id;
+        let extra_attachment = &tag.infos.extra_attachment;
+        if record_store_id != store_id || extra_attachment.is_empty() {
```

Comment:
`extra_attachment` is used by TiDB to embed SQL information.

Should PD care about whether the CPU load comes from SQL requests? If not, I think it doesn't need to skip these records.